Userland/Libraries/LibC/assert.h

```diff
@@ -11,7 +11,7 @@
 __BEGIN_DECLS
 
 #ifdef DEBUG
-[[noreturn]] void __assertion_failed(const char* msg);
+__attribute__((noreturn)) void __assertion_failed(const char* msg);
```

Comment:
weirdly enough [this](en.cppreference.com/w/cpp/language/attributes) says that `[[noreturn]]` is a standardised attribute

Reply:
Yes, it is standardized in C++.

However, it's [not yet](https://en.cppreference.com/w/c/language/attributes) part of the C standard. Most ports written in C did not care as GCC already has this as a "language extension". But technically speaking, that's not valid C code. Clang's `compiler-rt`, however, expects the C standard library to be standards-conformant, and failed to compile.